apps/challenges/admin.py

```diff
@@ -1,3 +1,28 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import Challenge, Phase
+
+
+@admin.register(Challenge)
+class ChallengeModelAdmin(TimeStampedAdmin):
+    list_display = ("title",
+                    "start_date",
+                    "end_date",
+                    "creator",
+                    "published",
+                    "enable_forum",
+                    "anonymous_leaderboard")
+    list_filter = ("creator",
+                   "published",
+                   "enable_forum",
+                   "anonymous_leaderboard")
+    search_fields = ("title", "creator")
+
+
+@admin.register(Phase)
+class PhaseModelAdmin(TimeStampedAdmin):
+    list_display = ("name", "leaderboard_public", "challenge")
+    list_filter = ("leaderboard_public", "challenge")
+    search_fields = ("name")
```

Comment:
search_fields needs to be a tuple or list. In python when creating a tuple with single element, it is necessary to insert comma at the last, like

```
search_fields=('name',)
````